#! /bin/sh /usr/share/dpatch/dpatch-run
## svn-display-sidebar.dpatch by Sebastien Bacher <seb128@ubuntu.com>
##
## DP: Display the sidebar by default (taken from upstream SVN)
## DP: LP: #286490
@DPATCH@
diff -urNad f-spot-0.5.0.3~/src/Preferences.cs f-spot-0.5.0.3/src/Preferences.cs
--- f-spot-0.5.0.3~/src/Preferences.cs	2008-10-16 08:42:01.000000000 +0200
+++ f-spot-0.5.0.3/src/Preferences.cs	2009-04-08 14:24:52.000000000 +0200
@@ -149,6 +149,7 @@
 				return (int) Tag.IconSize.Medium;
 		
 			case SIDEBAR_POSITION:
+				return 130;
 			case ZOOM:
 				return null;
 
